Solutions/SophosEP/Data Connectors/SophosEP_API_FunctionApp.json

```diff
@@ -0,0 +1,122 @@
+	"sampleQueries": [{
+		"description": "All logs",
+		"query": "SophosEP_CL\n | sort by TimeGenerated desc"
+	}
+],
+	"dataTypes": [{
+			"name": "SophosEP_CL",
+			"lastDataReceivedQuery": "SophosEP_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+		}
+	],
+	"connectivityCriterias": [{
+			"type": "IsConnectedQuery",
+			"value": [
+				"SophosEP_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+			]
+		}
+	],
+	"availability": {
+		"status": 1,
+		"isPreview": true
+	},
+	"permissions": {
+		"resourceProvider": [{
+				"provider": "Microsoft.OperationalInsights/workspaces",
+				"permissionsDisplayText": "read and write permissions on the workspace are required.",
+				"providerDisplayName": "Workspace",
+				"scope": "Workspace",
+				"requiredPermissions": {
+					"write": true,
+					"read": true,
+					"delete": true
+				}
+			},
+			{
+				"provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+				"permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+				"providerDisplayName": "Keys",
+				"scope": "Workspace",
+				"requiredPermissions": {
+					"action": true
+				}
+			}
+		],
+		"customs": [{
+				"name": "Microsoft.Web/sites permissions",
+				"description": "Read and write permissions to Azure Functions to create a Function App is required. [See the documentation to learn more about Azure Functions](https://docs.microsoft.com/azure/azure-functions/)."
+			},
+			{
+				"name": "REST API Credentials/permissions",
+				"description": "**API token** is required. [See the documentation to learn more about API token](https://docs.sophos.com/central/Customer/help/en-us/central/Customer/concepts/ep_ApiTokenManagement.html)"
+			}
+		]
+	},
+	"instructionSteps": [{
+			"title": "",
+			"description": ">**NOTE:** This connector uses Azure Functions to connect to the Sophos Central APIs to pull its logs into Azure Sentinel. This might result in additional data ingestion costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) for details."
+		},
+		{
+			"title": "",
+			"description": ">**(Optional Step)** Securely store workspace and API authorization key(s) or token(s) in Azure Key Vault. Azure Key Vault provides a secure mechanism to store and retrieve key values. [Follow these instructions](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) to use Azure Key Vault with an Azure Function App."
+		},
+		{
+			"description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**SophosEPEvent**](https://aka.ms/sentinel-SophosEP-parser) which is deployed with the Azure Sentinel Solution."
```

Comment:
As per the latest template there is change in additional requirement banner text so we have to update here as well could you please update it. You can take a reference from [here](https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_AzureFunctionApp_template/DataConnector_API_AzureFunctionApp_template.json)

Thanks,
Aditi Maheshwari